rel/i18n/emqx_schema.hocon

```diff
@@ -855,6 +855,21 @@ The default value 1.5 is following the MQTT 5.0 specification. This multiplier i
 mqtt_keepalive_multiplier.label:
 """Keep Alive Multiplier"""
 
+mqtt_keepalive_check_factor.desc:
```

Comment:
As discussed in slack, to make it easier to configure:
```suggestion
mqtt_keepalive_check_interval.desc:
```

and clamp the config value in the range of `[1000, (KaIntervalSeconds div 2) * 1000]`